_posts/en/pages/2016-01-18-segwit-wallet-dev.md

```diff
@@ -96,17 +95,13 @@ A wallet MUST implement all the features in this section, in order to be conside
 * Developers should be careful not to make an off-by-4-times mistake in fee estimation.
 
 
-#### Upgrade Safety
+#### Activation
```

Comment:
For the subheading changes here and at "Segwit native addresses (optional)", would you mind taking one of the following approaches to preserve the previous anchor (id) text so that people who link to, e.g., https://bitcoincore.org/en/segwit_wallet_dev/#upgrade-safety will still be directed to the appropriate section?:

1. Restore the previous heading text, or
2. Specify the old anchor id for the new subheads so that [Kramdown](https://kramdown.gettalong.org/syntax.html#specifying-a-header-id) will set the anchor id to that value.  E.g.,

```markdown
#### Activation {#upgrade-safety}
#### Segwit native addresses (optional) {#advanced-designs}
```